users/urls.py

```diff
@@ -3,7 +3,7 @@
 from users import views
 
 router = DefaultRouter()
-router.register(r'users', views.UserViewSet)
```

Comment:
Remove one view from urls, adding new one? I don't know what's going on in this commit ;)

Reply:
View class has changed name from UsersViewSet to UserViewSet (to split part for detail user and list of users)